utils/codegen/asn1ToC/docker/asn1c.Dockerfile

```diff
@@ -23,9 +23,9 @@ RUN apt-get update && \
 
 # install asnc1c
 WORKDIR /setup
-RUN git clone https://github.com/mouse07410/asn1c.git
+RUN git clone https://github.com/usdot-fhwa-stol/usdot-asn1c.git
```

Comment:
The CI results show that a docker image cannot be built using this Dockerfile. Please investigate